gems/sorbet/test/snapshot/partial/underscores/src/foo.rb

```diff
@@ -0,0 +1,7 @@
+
+class A
+  # This one tests hidden-definitions working
+  define_method(:foo) do |_, _|
+    puts 'hello, world!'
+  end
```

Comment:
hidden definitions uses a completely different codepath, and so it was working already. but I figured I'd add a test anyway